yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -202,7 +206,7 @@ def _probe_version(self):
 
     @property
     def available(self):
-        return self.basename is not None
+        return self._ffmpeg_location.get() or self.basename is not None
```

Comment:
patching something like this should be done in its own PR
```suggestion
        return self.basename is not None
```